### DISCUSSION TITLE
Sort users joined and parted alphabetically

### DISCUSSION THREAD

**Discussion by BushEdT** [Chatterino/chatterino2#3420] [Ideas]
Hi !

I recently enabled this feature and have noticed they were not sorted alphabetically, which would be a nice thing to find a specific user (way) faster

I do not think there are any downsides to that, nor would it be difficult to add 

What do you guys think ?

**Comment by zneix** [comment]
> I do not think there are any downsides to that

These messages you see in Chatterino are in fact a "report" of all JOIN/PART messages collected over last 60 seconds and every minute there're just posted for you in chat. So, the only factor that'd be lost is "order" of joined/parted the channel first/last.

I think this is something someone *could* find informative/useful and, personally, I don't see need to sort usernames there, however I see how you and others might find sorting usernames as a useful Qol improvement, so I could make a fix sorting usernames since it's not difficult to do.
Before I start doing anything I'd like to know what others think about it though.

**Reply by Felanbird** [reply]
I don't mind this change either, I can't think of any real reason why you would care about the current order, it's more or less seen as "random" and A-Z will always be useful for sorting.

**Comment by zneix** [comment]
Should be fixed once #3421 gets merged in - most likely this weekend. Then, you can download nightly build after the mentioned Pull Request gets merged and it will have my fix included :+1:

**Reply by Felanbird** [reply]
@BushedT As zneix mentioned yesterday, the changes were implemented earlier today and the nightly build can be found [here.](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build)

**Reply by BushEdT** [reply]
Thanks a lot! Did not actually think this would be implemented that fast.
I also noticed usernames are now clickable and coloured since 2.3.4 release, what a nice touch.

**Reply by Felanbird** [reply]
> Did not actually think this would be implemented that fast. 

Everyone just suddenly did Chatterino coding for Christmas 🤷
